src/core/importType.js

```diff
@@ -25,12 +25,15 @@ export function isBuiltIn(name, settings, path) {
 }
 
 function isExternalPath(path, name, settings) {
+  const modules = (settings && settings['import/external-modules']) || []
   const folders = (settings && settings['import/external-module-folders']) || ['node_modules']
 
   // extract the part before the first / (redux-saga/effects => redux-saga)
   const packageName = name.match(/([^/]+)/)[0]
 
-  return !path || folders.some(folder => -1 < path.indexOf(join(folder, packageName)))
+  return !path
+    || folders.some(folder => -1 < path.indexOf(join(folder, packageName)))
```

Comment:
no need for a yoda condition, and "includes" exists:
```suggestion
    || folders.some(folder => path.includes(join(folder, packageName)))
```